desihub/qlf
# Removing electron app bar

# How to test

Run `yarn electron` and it should open a window without header.

_You won't be able to drag the window as the code is not yet on the staging environment._
